_posts/en/posts/2016-01-24-clarification.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Clarifying Communications
+name: clarifying-communications
+type: post
+permalink: /en/2015/01/24/clarification
+share: true
+---
+The bitcoincore.org website is the official website for the Bitcoin Core project. Other sites may also host information about Bitcoin Core, but they are not under the control of the Bitcoin Core project. We know that the Bitcoin ecosystem can be confusing, and we are working hard to make these relationships more clear.
+
+For development work, Bitcoin Core mainly uses the `#bitcoin-core-dev` IRC channel on Freenode, [Github](https://github.com/bitcoin/bitcoin) and [the bitcoin-dev mailing list](http://lists.linuxfoundation.org/pipermail/bitcoin-dev/). We also recently created a community Slack group (request an invitation [here](https://slack.bitcoincore.org)) to enable the greater Bitcoin community to connect with Bitcoin Core developers and contributors more easily.
```

Comment:
@harding Core devs moderate the IRC channel and GitHub at least.

@TheBlueMatt Is it? I thought most (if not all) of the admins are not regular contributors to Core.